kolibri/core/assets/src/constants.js

```diff
@@ -198,3 +198,5 @@ export const Presets = Object.freeze({
 // This should be kept in sync with the value in
 // kolibri/core/exams/constants.py
 export const MAX_QUESTIONS_PER_QUIZ_SECTION = 50;
+// this is not used for the actual exam model
+export const MAX_QUESTION_OPTIONS_PER_QUIZ_SECTION = 500;
```

Comment:
I think we should just define this as a multiple of the constant above - either that, or it be dynamic depending on how many questions you are trying to select. But this is not required for string freeze.